### PR TITLE
Update troubleshoot-onboarding.md

### PR DESCRIPTION
**Pull request by msbemba**
Explicitly added details about Minimum requirements for Microsoft Defender for Endpoint

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7565

File: microsoft-365/security/defender-endpoint/troubleshoot-onboarding.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ ms.technology: mde
 You might need to troubleshoot the Microsoft Defender for Endpoint onboarding process if you encounter issues.
 This page provides detailed steps to troubleshoot onboarding issues that might occur when deploying with one of the deployment tools and common errors that might occur on the devices.
 
+Before you start troubleshooting issues with onboarding tools, it is important to check if the minimum reuqirements are met for onboarding devices to the services. [Learn about the licensing, hardware and software requirements to onboard devices to the service](minimum-requirements.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/troubleshoot-onboarding.md]:
```suggestion
Before you start troubleshooting issues with onboarding tools, it is important to check if the minimum requirements are met for onboarding devices to the services. [Learn about the licensing, hardware, and software requirements to onboard devices to the service](minimum-requirements.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**